keras/layers/convolutional.py

```diff
@@ -3409,6 +3409,12 @@ def compute_output_shape(self, input_shape):
     return tf.TensorShape([input_shape[0], length, input_shape[2]])
 
   def call(self, inputs):
+    if tf.not_equal(tf.size(inputs), 0) and sum(self.cropping) >= inputs.shape[1]:
```

Comment:
I think this check would be better as

if inputs.shape[1] is not None and sum(self.cropping) >= inputs.shape[1]:

You could still construct a tensor with size 0 and shape (None, None) that would cause this to crash.

Reply:
You are right, I did the change.

Thanks for the help!